src/univariate/continuous/skewnormal.jl

```diff
@@ -1,11 +1,23 @@
 """
-SkewNormal(ξ, ω, α)
-    The *skew normal distribution* is a continuous probability distribution
-    that generalises the normal distribution to allow for non-zero skewness.
+    SkewNormal(ξ, ω, α)
+
+The *skew normal distribution* is a continuous probability distribution that
+generalises the normal distribution to allow for non-zero skewness. Given a
+location `ξ`, scale `ω`, and shape `α`, it has the probability density function
+
+```math
+f(x; \\xi, \\omega, \\alpha) =
+\\frac{2}{\\omega \\sqrt{2 \\pi}} e^{-\\frac{(x-\\xi)^2}{2\\omega^2}}

```

Comment:
I think usually we use `exp`:
```suggestion
\\frac{2}{\\omega \\sqrt{2 \\pi}} \\exp{\\bigg(-\\frac{(x-\\xi)^2}{2\\omega^2}\\bigg)}
```